CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 ### Added
 
 - `sandstone/Alert` public class names `alert`, `content`, `fullscreen`, and `title`
+- `sandstone/PageViews` prop `css` to support customizing the pageviews styles
```

Comment:
I think we don't need to add this change log at this point because `publicClassName` has been `true` in the first place.
Instead of this, we need to address
- `aria-label`s of `PageViews.Page` and `QuickGuidePanels.Panel`
- `closeButtonAriaLabel` and `onClose` of `QuickGuidePanels`
are added.

Reply:
It's fixed.